eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Plutus/TxInfo.hs

```diff
@@ -218,23 +218,14 @@ transValidityInterval ::
   forall proxy era a.
   Inject (AlonzoContextError era) a =>
   proxy era ->
-  ProtVer ->
   EpochInfo (Either Text) ->
   SystemStart ->
   ValidityInterval ->
   Either a PV1.POSIXTimeRange
-transValidityInterval _ protVer epochInfo systemStart = \case
+transValidityInterval _ epochInfo systemStart = \case
```

Comment:
I'd rename this to `alonzoTransValidityInterval`, otherwise it's easy to accidentally use this function instead of the one defined in Conway.